index.html

```diff
@@ -2,7 +2,7 @@
 <html>
 <html>
     <head>
-    <link rel="stylesheet" type="text/css" href="styles.css">
+    <link rel="stylesheet" type="text/css" >
```

Comment:
This is a problem because: [External Stylesheet](http://www.w3schools.com/css/css_howto.asp).